test/reactors/BaseDutchOrderReactor.t.sol

```diff
@@ -349,27 +349,6 @@ abstract contract BaseDutchOrderReactorTest is PermitSignature, DeployPermit2, B
         quoter.quote(order.order, order.sig);
     }
 
-    // The input decays, which means the outputs must not decay. In this test, the
```

Comment:
Nice to remove this test in the base but would be nice to add one showing that both inputs&outputs decaying in the dutchV2ReactorTest